### ISSUE TITLE
[BUG] Removed round admins still count as admins in some cases

### ISSUE THREAD

**Issue by Powersource** [cobudget/cobudget#670]
If someone is a round admin and you remove them so they're no longer a member (participant) then they still have admin access to some things, such as being able to see unpublished buckets. This because we have code that looks like (bucketsPage resolver in this case)

```
      const currentMember = await prisma.roundMember.findUnique({
        where: {
          userId_roundId: {
            userId: user?.id ?? "undefined",
            roundId,
          },
        },
      });

      const isAdminOrGuide =
        currentMember && (currentMember.isAdmin || currentMember.isModerator);
```

Most thorough solution might be to change the last line to

```
currentMember?.isApproved && (currentMember.isAdmin || currentMember.isModerator);
```

in all cases where we have logic like this. But I think it might be enough (and better failsafe in case we forget the ?.isApproved in future logic) to set isAdmin and isModerator to false when removing a member. That would also make you less confused when re-adding a member, since they wouldn't be an admin by default.

**Comment by Powersource** [comment]
Should be fixed by 9d4e3eb0

**Comment by Powersource** [comment]
seems to be fixed on staging.
